src/services/errors.js

```diff
@@ -22,5 +22,9 @@ module.exports= {
 	refreshTokenIsInvalidError : {
 		name : 'refreshTokenIsInvalidError',
 		message : "Refresh token is invalid."
+	},
+	inviteIsAlreadyActiveError : {
```

Comment:
lets kindly use the convention in the file such that they start with uppercase letters. 

didn't notice it for the refreshToken error as well but yea let's kindly keep the naming consistent

Reply:
yes please